futurenhs.api/FutureNHS.Api/Services/GroupService.cs

```diff
@@ -411,7 +414,7 @@ public async Task<(uint, IEnumerable<GroupMember>)> GetGroupMembersAsync(Guid us
             return await _groupDataProvider.GetGroupMembersAsync(slug, offset, limit, sort, cancellationToken);
         }
 
-        public async Task<(uint, IEnumerable<PendingGroupMember>)> GetPendingGroupMembersAsync(Guid userId, string slug, uint offset, uint limit, string sort, CancellationToken cancellationToken)
+        public async Task<(uint totalPending, IEnumerable<PendingGroupMember>)> GetPendingGroupMembersAsync(Guid userId, string slug, uint offset, uint limit, string sort, CancellationToken cancellationToken)
         {
             if (Guid.Empty == userId) throw new ArgumentOutOfRangeException(nameof(userId));
             if (string.IsNullOrWhiteSpace(slug)) throw new ArgumentOutOfRangeException(nameof(slug));
```

Comment:
var userCanPerformAction = await _permissionsService.UserCanPerformActionAsync(userId, slug, GroupViewRole, cancellationToken); 

I know you didn't write this but can the role be changed to GroupEditRole please as GroupView is not correct